[DOCS-6151] Update CCM Tag Pipelines docs (#19620)
DataDog/documentation: content/en/cloud_cost_management/_index.md

## Tag pipelines

You can use [tag pipelines][1] to create tag rules to help fix missing or incorrect tags on your Cloud bill, or to create new, inferred tags that align with business logic.

### Rule types

<div class="alert alert-warning"> A maximum of 100 rules can be created, and API based Reference Tables are not supported. </div>

> There are three types of rules supported: **Add tag**, **Alias tag keys**, and **Map multiple tags**. You can keep your rules organized by leveraging rules-sets, which act as folders for your rules. The rules are executed in order (from top to bottom), to keep the execution order deterministic. You can organize rules and rulesets to ensure the order of execution matches your business logic.

Reviewer: ```suggestion
There are three types of rules supported: **Add tag**, **Alias tag keys**, and **Map multiple tags**. You can keep your rules organized by leveraging rules sets, which act as folders for your rules. The rules are executed in deterministic order (from top to bottom). You can organize rules and rule sets to ensure the order of execution matches your business logic.
```

> **Add tag** - Add a new tag (key + value) based on the presence of existing tags on Cloud Costs data. 

Reviewer: ```suggestion
#### Add tag

Add a new tag (key + value) based on the presence of existing tags on Cloud Costs data. 
```

For example, you can create a rule to tag all resources with their business unit based on the services those resources are a part of. 
{{< img src="cloud_cost/tags_addnew.png" alt="Add new business unit tag to resources with service:processing, service:creditcard, or service:payment-notification." >}}

> **Alias tag keys** - Map existing tag values to a more standardized tag. 

Reviewer: ```suggestion
#### Alias tag keys

  Map existing tag values to a more standardized tag. 
```

For example, if your organization wants to use the standard `application` tag key, but several teams have a variation of that tag (like `app`, `webapp`, or `apps`), you can alias `apps` to `application`. Each alias tag rule allows you to alias a maximum of 25 tag keys to a new tag.

The rule stops executing for each resource after a first match is found. For example, if a resource already has a `app` tag, then the rule no longer attempts to identify a `webapp` or `apps` tag.
{{< img src="cloud_cost/tags_alias.png" alt="Add application tag to resources with app, webapp, or apps tag." >}}

> **Map multiple tags** - Use [Reference Tables][2] to add multiple tags to cost data without needing to create multiple rules. This will map the values from your Reference Table's primary key column to values from cost tags. If found, it will add the selected Reference Table columns as tags to cost data.

Reviewer: By "it" do you mean the pipeline or the Datadog application?
```suggestion
#### Map multiple tags

Use [Reference Tables][2] to add multiple tags to cost data without creating multiple rules. This will map the values from your Reference Table's primary key column to values from cost tags. If found, the pipelines adds the selected Reference Table columns as tags to cost data.
```
